packages/kit/src/views/Swap/components/LimitOrderCard.tsx

```
@@ -249,23 +258,34 @@ const LimitOrderCard = ({
-          <IconButton
-            icon="DeleteOutline"
-            variant="tertiary"
-            color="$iconSubdued"
-            size="small"
-            onPress={() => {
+          <Badge
+            px="$2"
+            bg="$bgSubdued"
+            borderRadius="$2.5"
+            borderWidth={1}
+            borderColor={'$borderSubdued'}
```

ESLint

[error] Curly braces are unnecessary here.

(react/jsx-curly-brace-presence)